tasks/11_find-longest-word/find-longest-word.js

```diff
@@ -0,0 +1,6 @@
+function findLongestWord(sentence) {
+  let callback = (max, current) => Math.max(max, current);
+  return sentence.split(' ').map((s) => s.length).reduce( callback );
```

Comment:
Pls use more meaningful names for variables, params, etc.

And also, do we really need callback as separate variable?
In real-world it would lead to spaghetti-code.